### ISSUE TITLE
Training - Hyper-parameters

### ISSUE THREAD

**Issue by balancap** [Zehaos/MobileNet#13]
Hello,

thanks a lot for releasing the implementation and pre-trained weights of MobileNets!
I am trying as well to reproduce the accuracy reported in the paper, and I have a bit of trouble finding the optimal hyper-parameters for training. What is your experience on the subject? Have any tip? :)

Thanks!

**Comment by Zehaos** [comment]
Hi,
I used command below， run for about 1000K step to produce the result(66.51% top-1 accuracy).
It is likely to get a better result if the training is fully finished.
```
python train_image_classifier.py \
  --train_dir=${TRAIN_DIR} \
  --dataset_name=imagenet \
  --dataset_split_name=train \
  --dataset_dir=${DATASET_DIR} \
  --model_name=mobilenet \
  --preprocessing_name=mobilenet \
  --width_multiplier=1.0 \
  --max_number_of_steps=2000000 \
  --batch_size=128 \
  --save_interval_secs=240 \
  --save_summaries_secs=240 \
  --log_every_n_steps=100 \
  --optimizer=rmsprop \
  --rmsprop_decay=0.9 \
  --opt_epsilon=1.0\
  --learning_rate=0.1 \
  --learning_rate_decay_factor=0.1 \
  --momentum=0.9 \
  --num_epochs_per_decay=30.0 \
  --weight_decay=0.0 \
  --num_clones=2
```

In my experience, 
- Use large batch size, should not be smaller than 128, which is also mentioned in ResNet paper.
- Use smaller '--num_epochs_per_decay'.

Some suggestions, 
- Use less data augmentation.
- Use small 'weight_decay', (eg. 1e-4).

Some successful examples
- Use small weight decay (1e-4), linear lr policy, inception data augmentation. [MobileNet-Caffe Question for training details #1](https://github.com/shicai/MobileNet-Caffe/issues/1)

**Comment by Zehaos** [comment]
A slight digression，
I have integrated MobileNet into [SSD-Tenroflow](https://github.com/balancap/SSD-Tensorflow), and I trained it, but the loss is not seems to decreased. And I noticed that the training part is still have some problems. Will it be fixed in the future? Can I help to do something? Thanks.

**Comment by Zehaos** [comment]
SSD-Tenroflow had been fixed. Please refer to [SSD_tensorflow_VOC](https://github.com/LevinJ/SSD_tensorflow_VOC).
